core/src/main/proto/Locator.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+//option java_multiple_files = true;
+package core.src.edu.rice.comp413fall2020grey;
+import "google/protobuf/timestamp.proto";
+
+// Request payload
+message ServerInformation {
+  // Each message attribute is strongly typed.
+  // You also must assign a "tag" number.
+  // Each tag number is unique within the message.
+  string server_name = 1;
+
+  // This defines a strongly typed list of String
+  string server_type = 2;
+
+  // There are many more basics types, like Enum, Map
+  // See https://developers.google.com/protocol-buffers/docs/proto3
+  // for more information.
+}
+
+message BasicInfo {
+  string senderId = 1;
+  google.protobuf.Timestamp time = 2;
+}
+
+message ConnectionConfirmation {
+  string confirmation = 1;
+}
+
+message QueryRegistrar {
+  // BSON Query String
+  string query = 1;
+  BasicInfo info = 2;
+}
+
+message QueryResponse {
```

Comment:
I dont think we need any of these messages any more. Instead you should add any necessary messages to the main proto file